src/frequenz/sdk/microgrid/metadata.py

```diff
@@ -0,0 +1,54 @@
+# License: MIT
+# Copyright © 2023 Frequenz Energy-as-a-Service GmbH
+
+"""Metadata that describes a microgrid."""
+
+from dataclasses import dataclass
+from zoneinfo import ZoneInfo
+
+from timezonefinder import TimezoneFinder
+
+_timezone_finder = TimezoneFinder()
+
+
+@dataclass
```

Comment:
Any reason not to use `kw_only=True, frozen=True`?

Reply:
No, no reason after using the trick to set the attribute of a frozen dataclass. 
It is already updated as suggested